Optimiser.fs

```diff
@@ -742,6 +797,7 @@ module Graph =
                      [ ("graph-collapse-nops", true, collapseNops)
                        ("graph-collapse-ites", true, collapseITEs)
                        ("graph-drop-local-edges", true, dropLocalEdges model.Locals)
+                       ("graph-collapse-unobservable-edges", true, collapseUnoberservableEdges model.Locals)
```

Comment:
the behaviour is the same as @CaptainHayashi had originally (hopefully!)

As far as I understand, you can imagine these booleans as the "default" behaviour (enabled if true/disabled if false) and the command line arguments override this default.